https/users.go

```diff
@@ -51,19 +51,25 @@ func (u *userAuthRoundtrip) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 
 	if len(c.Users) == 0 {
+		w.Header().Add("Strict-Transport-Security", "max-age=63072000; includeSubDomains")
+                w.Header().Add("X-Frame-Options", "SameOrigin")
```

Comment:
This code is going to be reused by other systems such as Prometheus, where we do expect things to be served inside an iframe.